README.md

```diff
@@ -44,20 +44,14 @@ import Editor from 'react-simple-code-editor';
 import { highlight, languages } from 'prismjs/components/prism-core';
 import 'prismjs/components/prism-clike';
 import 'prismjs/components/prism-javascript';
+import 'prismjs/themes/prism.css' //Example style
 
-const code = `function add(a, b) {
-  return a + b;
-}
-`;
-
-class App extends React.Component {
-  state = { code };
-
-  render() {
+function App(){
+    const [code,setCode] = React.useState(`function add(a, b) {return a + b; }`);
```

Comment:
can you format the code missing spaces), `[code,setCode]`, `function App() {`

Reply:
even `function add(a, b) {\n  return a + b;\n}` too

Reply:
fix indentation too `    const [code,setCode]` should be 1 indent level, 2 spaces

Reply:
@caub 
Okey, I'll fix it later.

Reply:
@caub 
I have fixed codestyle.
Can you merge this PR, please?